### PR TITLE
Features/arcticspacefox/lotto

### PR DESCRIPTION
**Pull request by ArcticSpaceFox**
# Lotto feature

will handle all lotto stuff, like buying tickets, eventually winning

### CODE REVIEW

**Comment by ArcticSpaceFox** [comment]
Ok only missing thing is to send a message to a channel which the winners will be send to. Also checking if everything works

File: index.js [hunk 1]
```diff
@@ -79,6 +87,58 @@ class EzField {
...
+
+  // reset Lotto db
+  await prisma.lottoTicket.deleteMany({});
+
+  // send winner message
+  // TODO: send message
+  console.log(`(lotto) : This weeks winning numbers are : ${winningNumbers} | Winners: ${winners.length}`);
+
+}, 1000*60*60*24*7);
+
```
Comment (ryzetech) [inline index.js:141]: probably unsafe due to seperate execution

File: index.js [hunk 2]
```diff
@@ -1320,11 +1442,95 @@ client.on('message', async (message) => {
...
+    // Check if numbers are not the same
+    const numberset = new Set(numbers);
+    const duplicates = numbers.filter(n => !numberset.has(n))
+    if (numberset.size !== 5) {
+      return message.channel.send(new Discord.MessageEmbed()
+        .setColor(embedColorFail)
+        .setAuthor("Lotto System", embedPB)
+        .setTitle("❌ Syntax mistake! Duplicate numbers found")
+        .setDescription("Sytntax is `#lotto buy <number1> <number2> <number3> <number4> <number5>`")
+        .addField("`number1..5`:", "The numbers you choose for your lotto ticket. Can be between `01` to  `50`. Notice a number can only be choosen **once**!")
```
Comment (ryzetech) [inline index.js:1483]: inconsistant random range display

**Review by ryzetech — COMMENTED** [review]
some things still need to be fixed, otherwise great work! 👍

File: index.js (hunk 1 above)
Comment (ArcticSpaceFox) [reply index.js:141]: nope its async

Comment (ArcticSpaceFox) [reply index.js:141]: ohh ahh ok ahh, yes i wanna put it in your loop thingy

**Comment by ryzetech** [comment]
We are postponing the feature for now. There are too many collisions and too many things that need to be fixed. Appreciate the work tho!

**Pull request closed without merging**